.changeset/chilled-bats-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ciena-org/ember-codemods": minor
```

Comment:
Can you make this a patch instead? The package is still in the alpha release, so bugs and features are patch and breaking changes will be minor. This is specified [here](https://github.com/ciena-blueplanet/ember-codemods/blob/main/CONTRIBUTING.md?plain=1#L75)
```suggestion
"@ciena-org/ember-codemods": patch
```